vars/bayesianScanner.groovy

```diff
@@ -14,11 +14,10 @@ def call(body) {
         echo "Checking ${serviceName} exists"
         if (flow.hasService(serviceName)) {
             try {
-                sh '''
-                    mvn io.github.stackinfo:stackinfo-maven-plugin:0.2:prepare 
-                    mvn org.apache.maven.plugins:maven-dependency-plugin:3.1.1:collect -DoutputFile=direct-dependencies.txt -DincludeScope=runtime -DexcludeTransitive=true 
-                    mvn org.apache.maven.plugins:maven-dependency-plugin:3.1.1:collect -DoutputFile=transitive-dependencies.txt -DincludeScope=runtime -DexcludeTransitive=false
-                   '''
+                sh "#!/bin/bash \n" +
```

Comment:
trivial: isn't 
```
sh """#/bin/bash 

   mvn io.github.stackinfo:.... 
"""
```

better?

Reply:
@sthaha I tried this and it did not work here is the supporting document
https://support.cloudbees.com/hc/en-us/articles/215171038-How-do-I-run-a-different-Shell-interpreter-in-a-Pipeline-Script-

Reply:
I am not sure if we have a better approach than this, but this works and we do not have to do any modification in old Jenkinsfiles.